executor/ddl.go

```diff
@@ -67,6 +67,11 @@ func (e *DDLExec) Next(ctx context.Context, chk *chunk.Chunk) (err error) {
 		return nil
 	}
 	e.done = true
+
+	// For every DDL, we must commit current transaction.
+	if err := e.ctx.NewTxn(); err != nil {
```

Comment:
I am confused, according to the comments, i am expecting to call something like "Commit".